### PR TITLE
Added flag to handle responsive tables

### PR DESCRIPTION
**Pull request by zone31**

### CODE REVIEW

File: lib/api_maker/javascript/bootstrap/card.jsx [hunk 1]
```diff
@@ -54,7 +55,9 @@ export default class Card extends React.Component {
     var classNames = ["card-body"]
 
     if (this.props.table) {
-      classNames.push("table-responsive")
+      if(this.props.responsiveTable){
```
Comment (kjabtion) [inline lib/api_maker/javascript/bootstrap/card.jsx]:
```suggestion
      if (this.props.responsiveTable){
```

File: lib/api_maker/javascript/bootstrap/card.jsx [hunk 2]
```diff
@@ -54,7 +55,9 @@ export default class Card extends React.Component {
...
+        classNames.push("table-responsive")
+      }
```
Comment (kjabtion) [inline lib/api_maker/javascript/bootstrap/card.jsx]:
```suggestion
      }

```

File: lib/api_maker/javascript/bootstrap/card.jsx [hunk 3]
```diff
@@ -11,6 +11,7 @@ export default class Card extends React.Component {
     onClick: PropTypes.func,
     striped: PropTypes.bool,
     style: PropTypes.object,
+    responsiveTable: PropTypes.bool,
```
Comment (kjabtion) [inline lib/api_maker/javascript/bootstrap/card.jsx]: This was default before. Not it wont be. For backwards compatibility you could add some defaultProps like this:

```jsx
static defaultProps = {
  responsiveTable: true
}
```

Comment (zone31) [reply lib/api_maker/javascript/bootstrap/card.jsx]: Sure, will do :)

**Comment by zone31** [comment]
I think everything should be up to shape now :)

**Pull request merged**